deployment/network-operator/values.yaml

```diff
@@ -32,28 +33,37 @@ sriovNetworkOperator:
 
 # Node Feature discovery chart related values
 node-feature-discovery:
-  image:
-    pullPolicy: IfNotPresent
-  nodeFeatureRule:
-    createCRD: false
-  master:
-    instance: "nvidia.networking"
```

Comment:
IIRC this is needed to support several deployments of NFD.
i dont think we can remove it until we start using NodeFeature (creating the CR)

@e0ne @ykulazhenkov  thoughts ?

Reply:
as a note:

@ArangoGutierrez  in the latest PR version now keeps this configuration.

i re-opened this comment  to get some additional feedback on this bit while at it.

but should not affect merging this PR.

Reply:
I can't say if it is better to keep this config or not. We need to think/test the upgrade scenario to choose the correct default.